tests/System.Text.Primitives.Tests/Encoding/EncodeIntoUtf8Tests.cs

```diff
@@ -4,8 +4,10 @@
 
 using System.Buffers;
 using System.Buffers.Text;
+using System.Runtime.InteropServices;
 using Xunit;
 
+
```

Comment:
nit: remove empty line

Reply:
the tests could use some minor code formatting cleanup (sorting using statements, fixing up all the deferred TODOs, removing blank lines, etc).  Let's fix those separately from the fixup for the breaking API change.